pkg/jx/cmd/vault/interfaces.go

```diff
@@ -0,0 +1,17 @@
+package vault
```

Comment:
I would move this file to `pkg/vault`. We should keep all the vault specific things in that package.